NCI-GDC/gdc-frontend-framework
# Sjpp create cohort

## Description
Use the CreateCohortModal in ProteinPaint-based tools, so that the displayed cohort pill will be collapsed to the set_id name, instead of listing many case_ids.

To test manually:
1. Oncomatrix: click and drag to select case columns, a menu will open on mouseup, select `Create Cohort`, the modal will show up.
2. ProteinPaint: type in a gene, click on a matching result, click on the `# samples` on the left side, a sample table will popup. Check to select samples in the second column, then click `Create Cohort` in the bottom right.

## Checklist

      flagged issues

## Screenshots/Screen Recordings (if Appropriate)

![Screenshot 2023-11-14 at 9 55 15 PM](https://github.com/NCI-GDC/gdc-frontend-framework/assets/411031/51d0b298-3019-4fff-a98f-3448dd207f72)
